service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/PlacementSubmittedControllerTest.java

```diff
@@ -55,20 +54,22 @@
     private static final String RESPONDENT_SURNAME = "Watson";
     private static final String LOCAL_AUTHORITY_CODE = "example";
     private static final String SEND_DOCUMENT_EVENT = "internal-change:SEND_DOCUMENT";
-
-    private final DocumentReference documentReference = testDocument();
+    private static final String JURISDICTION = "PUBLICLAW";
```

Comment:
I think this should be a case constant accessible from anywhere so no need to re define